spec/examples.txt

```diff
@@ -0,0 +1,29 @@
+example_id                                 | status | run_time        |
+------------------------------------------ | ------ | --------------- |
+./spec/integration/post_index_spec.rb[1:1] | passed | 0.34439 seconds |
+./spec/integration/post_index_spec.rb[1:2] | passed | 0.3891 seconds  |
+./spec/integration/post_index_spec.rb[1:3] | passed | 0.40858 seconds |
+./spec/integration/post_index_spec.rb[1:4] | passed | 0.38001 seconds |
+./spec/integration/post_index_spec.rb[1:5] | passed | 0.39885 seconds |
+./spec/integration/post_index_spec.rb[1:6] | passed | 0.59555 seconds |
+./spec/integration/post_index_spec.rb[1:7] | passed | 0.39248 seconds |
+./spec/integration/post_index_spec.rb[1:8] | passed | 0.35838 seconds |
+./spec/integration/post_index_spec.rb[1:9] | passed | 0.50485 seconds |
+./spec/integration/post_show_spec.rb[1:1]  | passed | 0.32598 seconds |
+./spec/integration/post_show_spec.rb[1:2]  | passed | 0.34619 seconds |
+./spec/integration/post_show_spec.rb[1:3]  | passed | 2.05 seconds    |
+./spec/integration/post_show_spec.rb[1:4]  | passed | 0.35963 seconds |
+./spec/integration/post_show_spec.rb[1:5]  | passed | 0.34752 seconds |
+./spec/integration/post_show_spec.rb[1:6]  | passed | 0.49031 seconds |
+./spec/integration/user_index_spec.rb[1:1] | passed | 0.38947 seconds |
+./spec/integration/user_index_spec.rb[1:2] | passed | 0.32507 seconds |
+./spec/integration/user_index_spec.rb[1:3] | passed | 0.56367 seconds |
+./spec/integration/user_index_spec.rb[1:4] | passed | 1.72 seconds    |
+./spec/integration/user_show_spec.rb[1:1]  | passed | 0.52777 seconds |
+./spec/integration/user_show_spec.rb[1:2]  | passed | 0.85807 seconds |
+./spec/integration/user_show_spec.rb[1:3]  | passed | 0.85252 seconds |
+./spec/integration/user_show_spec.rb[1:4]  | passed | 0.45515 seconds |
+./spec/integration/user_show_spec.rb[1:5]  | passed | 0.56773 seconds |
+./spec/integration/user_show_spec.rb[1:6]  | passed | 0.45425 seconds |
+./spec/integration/user_show_spec.rb[1:7]  | passed | 0.51213 seconds |
+./spec/integration/user_show_spec.rb[1:8]  | passed | 0.44719 seconds |
```

Comment:
- [ **OPTIONAL** ] Good job as all tests for this milestone are passing. But I think it would be better if you fix some of your previous tests which are failing to make your application more professional. Good job :+1: again.